datafusion/functions-aggregate/src/min_max.rs

```diff
@@ -318,6 +320,18 @@ impl AggregateUDFImpl for Max {
             Timestamp(Nanosecond, _) => {
                 primitive_max_accumulator!(data_type, i64, TimestampNanosecondType)
             }
+            Duration(Second) => {
```

Comment:
I think you need to add `Duration` to the list of types supported in `groups_accumulator_supported` as well.